### PR TITLE
CONN-800:Fix unitTests

### PR DESCRIPTION
**Pull request by alameluchidambaram**

### CODE REVIEW

File: Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/connectmgr/ConnectionManagerCacheTest.java [hunk 1]
```diff
@@ -38,22 +38,36 @@
...
  * @author Arthur Kong, msw
  */
 public class ConnectionManagerCacheTest extends BaseConnctionManagerCache {
 
+    private PropertyAccessor accessor;
+
+    @Before
+    public void setUp() {
+    }
+
```
Comment (jsmith2) [inline Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/connectmgr/ConnectionManagerCacheTest.java]: Why is this empty before method included?

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/asyncmsgs/dao/AsyncMsgRecordDao.java [hunk 2]
```diff
@@ -615,6 +611,38 @@ public void checkExpiration() {
...
+            value = accessor.getPropertyLong(NhincConstants.GATEWAY_PROPERTY_FILE,
+                    NhincConstants.ASYNC_DB_REC_EXP_VAL_PROP);
+        } catch (PropertyAccessException ex) {
+            LOG.error("Error: Failed to retrieve " + NhincConstants.ASYNC_DB_REC_EXP_VAL_PROP + " from property file: "
+                    + NhincConstants.GATEWAY_PROPERTY_FILE);
+            LOG.error(ex.getMessage());
+        }
+        return value;
+    }
+
```
Comment (jsmith2) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/asyncmsgs/dao/AsyncMsgRecordDao.java:645]: I think the above 2 methods should be private.

Comment (alameluchidambaram) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/asyncmsgs/dao/AsyncMsgRecordDao.java:645]: I can change it to private.

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/util/HomeCommunityMap.java [hunk 3]
```diff
@@ -272,18 +281,19 @@ public static String getHomeCommunityIdWithPrefix(String communityId) {
...
-    
+
     /**
      * Used for injecting mock property accessor for Unit testing.
+     * 
      * @param propAccessor
      */
-    protected static void setPropertyAccessor(PropertyAccessor propAccessor){
-        propertyAccessor = propAccessor;
+    public static void setPropertyAccessor(PropertyAccessor propAccessor) {
```
Comment (jsmith2) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/util/HomeCommunityMap.java]: why is the connectionmanagercache injected by constructor and the propertyaccessor is injected by setter.  Should add propertyaccessor to constructor and remove this setter.

Comment (alameluchidambaram) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/util/HomeCommunityMap.java]: Some of the methods that access the PropertyAccessor are static method and keeping this static makes it easier.

File: Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/util/HomeCommunityMapTest.java [hunk 4]
```diff
@@ -46,25 +47,24 @@
 import org.jmock.Mockery;
```
Comment (msweaver) [inline Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/util/HomeCommunityMapTest.java]: I think these jmock imports should be unused at this point. Can you remove them?

Comment (alameluchidambaram) [inline Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/util/HomeCommunityMapTest.java]: yes. Removed unused imports.

**Pull request merged**